packages/amplication-git-utils/src/git/git.service.ts

```diff
@@ -124,6 +125,19 @@ export class GitClientService {
     }
   }
 
+  private async createBranchIfNotExists(args: OneBranchArgs) {
```

Comment:
Why did you add this method to the git provider?
@yuval-hazaz when we talked about it while I was working on the refactoring we decided that those functions will be in the provider and not in the git service. What changed

Reply:
https://github.com/amplication/amplication/pull/5128

Reply:
I think this way is ok